### PR TITLE
Anthony - Task: Create a profile UI

### PR DESCRIPTION
**Pull request by a-iacampo**
# Description
- Added a header and a taskbar 
- Added a user profile card to display user info 
- Added a friends (followers and following) and photos page
- Implemented React Router
- Pull user profile information from Firebase

### CODE REVIEW

File: picturepoint/src/Layouts/Taskbar.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const useStyles = makeStyles({
+    root: {
+      flexGrow: 1,
+    },
+});
+
+//Render
+function Taskbar(props) {
+    const classes = useStyles();
+    const [value, setValue] = useState(0); //<- BUG here
```
Comment (Eltom777) [inline picturepoint/src/Layouts/Taskbar.jsx:21]: why is there a bug ? What needs to be fixed ?

**Review by Eltom777 — CHANGES_REQUESTED** [review]
why is there a bug ?

File: picturepoint/src/Layouts/Taskbar.jsx (hunk 1 above)
Comment (a-iacampo) [reply picturepoint/src/Layouts/Taskbar.jsx:21]: Currently, the taskbar is set to a default value of 0 which will set it to "My Friends". However, this creates a bug where refreshing the page always sets the taskbar back to "My Friends". I need to create a variable to keep track of the current page so refreshing doesn't change the taskbar.

Comment (Eltom777) [reply picturepoint/src/Layouts/Taskbar.jsx:21]: perfect ! thanks for the short demo on discord 💯

**Comment by Eltom777** [comment]
bug is tagged to #34

**Review by Eltom777 — APPROVED** [review]

**Review by Jordan-Hum — APPROVED** [review]
good job 🤠

**Comment by Eltom777** [comment]
🥇

**Pull request merged**